webknossos-tracingstore/app/com/scalableminds/webknossos/tracingstore/annotation/TSAnnotationService.scala

```diff
@@ -60,6 +60,8 @@ class TSAnnotationService @Inject()(val remoteWebknossosClient: TSRemoteWebknoss
   private lazy val materializedAnnotationWithTracingCache =
     AlfuCache[ObjectId, AlfuCache[Long, AnnotationWithTracings]](maxCapacity = 1000)
 
+  private val MAX_UPDATE_ACTION_ENTRIES_PER_VERSION = 1000
```

Comment:
I’d suggest to move this line into the function itself so the scope becomes clearer. Otherwise with this name, I’d assume that this affects a lot of stuff in this service. Also https://docs.scala-lang.org/style/naming-conventions.html#constants-values-and-variables says that “Constant names should be in upper camel case”

Reply:
Oh didn't know about this convention :D